core/core-workflow-steps/src/test/resources/CsvAnalyzerTest/dependencies.csv

```diff
@@ -1,2 +1,2 @@
-Artifact Id,Group Id,Version,License Short Name,License Long Name,File Name,Coordinate Type,Observed License Short Name,Observed License Long Name,Source URL,Release Tag URL,Software Heritage URL
-commons-csv,org.apache.commons,1.4,Apache-2.0,Apache Software License 2.0,commons-csv.jar,mvn,Apache-2.0,Apache Software License 2.0,http://archive.apache.org/dist/commons/csv/source/commons-csv-1.4-src.zip,https://github.com/apache/commons-csv/tree/csv-1.4,archive.apache.org/dist/commons/csv/source/commons-csv-1.4-src.zip.sha1
+Artifact Id,Group Id,Version,License Short Name,License Long Name,File Name,Coordinate Type,Observed License Short Name,Observed License Long Name,Source URL,Release Tag URL,Software Heritage ID
```

Comment:
Would it be okay to move those changes to the already open issue about adaptions in the CSV Analyzer? Since this is primarily a PR for sw360 functionality. 
The changes here have only been made due to changes in the model of artifact facts

Reply:
Sounds okish, since I already said that this PR is pretty big.

Reply:
> Why do we have License long names here? We need to reference the licenses, the id is therefore good enough, but we need the expression on how the licenses are related!

We need the long name if it is a license which is not known to the knowledgebase.

Reply:
How is this saved then? The component does not provide the means to store that. Is a new license created in this case? I think we could separate this into license creation and component creation.

Reply:
Our artifacts in the antenna execution contain facts with content `License extends LicenseInformation`. These contain the following fields:
- name
- longName
- text
- threatGroup
- classification.

We do not actually create licenses shared between artifacts in an antenna execution.

Reply:
I had too much the SW360 view in my mind. You are right, we can handle licenses in Antenna runs when we use it for document generation. But I doubt that transporting a license text in CSV is no fun.